app/views/layouts/_header.html.erb

```diff
@@ -23,6 +23,9 @@
           <li class="nav-item">
             <%= link_to "Getting Started", faq_path, class: 'nav-link', "data-turbolinks": "false" %>
           </li>
+          <li class="nav-item">
+            <%= link_to "Trent Mode", trent_mode_path(network: params[:network]), class: 'nav-link', "data-turbolinks": "false" %>
```

Comment:
do we have to turn off turbolinks here ?

Reply:
We don't. Good catch